plugins/v1/repository_s3/README.md

```diff
@@ -0,0 +1,28 @@
+This directory contains the (optional) keystore configuration for the `repository-s3` plugin.
+For more details on secure settings for the repository-s3 plugin please refer to the [repository-s3-client](https://www.elastic.co/guide/en/elasticsearch/plugins/7.5/repository-s3-client.html) documentation.
+
+### Parameters
+
+This configuration allows to set the following parameters with Rally using `--plugin-params` in combination with `--elasticsearch-plugins="repository-s3"`:
+
+* `aws_access_key`: A string specifying the AWS access key (mandatory)
```

Comment:
You’ve named the plugin properties `aws_*` and there is another AWS related plugin for discovery so I wonder if one would ever want to use different credentials? How about `s3_*`?

Reply:
This's been fixed in https://github.com/elastic/rally-teams/pull/39/commits/2c8a757e0b301bc10895142731bbb309dff9773c